Dream-Home-PEABOT/FEE_DreamHome
# Om feature display survey

### What does this PR do?
- This PR creates a Survey component 
- It styles the Survey component following the wireframe guideline

### Where should the reviewer start?
-  `src/components/Survey/Survey.css`
- `src/components/Survey/Survey.tsx`
-
### How should this be manually tested?
- Run `npm start`, the application should start on `localhost/3000`
- Scroll down to the Survey component, the component should match the wireframe provided by the team
<img width="1435" alt="Screen Shot 2021-01-06 at 10 35 50 PM" src="https://user-images.githubusercontent.com/56229864/103855255-89e4d780-506f-11eb-9699-86f4979b897b.png">

### Any background context you want to provide?

### What are the relevant tickets?

- closes #31 

### Screenshots (if appropriate)

<img width="1439" alt="Screen Shot 2021-01-06 at 10 38 19 PM" src="https://user-images.githubusercontent.com/56229864/103855417-e1834300-506f-11eb-8441-7961fb0f5aa6.png">
